documents/Libraries/README.md

```diff
@@ -32,4 +36,17 @@ The following is the layout of the definitions and implementations provided as p
         - [pbrlib_genosl_impl.mtlx](pbrlib/genosl/pbrlib_genosl_impl.mtlx) : Mapping from definitions to implementations
         -   OSL implementation files.
 
-Note that there is no reference implementation for the PBR library.
+## Support Notes:
+- GLSL language support is for version 4.0 or higher.
+- "default" color management support includes OSL and GLSL implementations for the following non-LUT transforms:
+    - lin_rec709, gamma18, gamma22, gamm24, acescg, srgb_texture
+- Basic GLSL `lightshader` node definitions and implementations are provided for the following light types:
+    - point, directional, spot
+- Code generation is not currently supported for:
+    - `noise` nodes for `genglsl`
+    - `ambientocclusion` node for: `genosl` and `genglsl`.
+    - `arrayappend` node for: `genosl` and `genglsl`.
+    - `curveadjust` node for: `genosl` and `genglsl`.
+    - `displacementshader` and `volumeshader` nodes and associated operations (`add`, `multiply`, `mix`) for: `genosl` and `genglsl`.
+    - `add`, `multiply`, and `mix surfaceshader` node operations for `genglsl`.
+    - `PBR` library support in the reference `osl` implementation.
```

Comment:
Not sure we need this last point. It's not "currently unsupported" as in _will be added later_. The genosl and genglsl implementations of pbrlib is kind of the reference.